### DISCUSSION TITLE
Is there an autobrr API I can use? Is there a way to use autobrr to send to my seedbox torrents fetched by my local script, without setting up an indexer?

### DISCUSSION THREAD

**Discussion by ikki4** [autobrr/autobrr#1151] [General]
I know this sounds very confusing, so I'll explain a little better.

There's this UNIT3D tracker I'm in, that has no IRC announce (so it has no autobrr indexer). This tracker provides RSS and API access. I prefer using the API because it brings me more information than RSS, so I can better customize my filters in my script.

The problem is, I haven't been able to use my script to send torrents to my seedbox's qBittorrent. I can send them to my local qBittorrent, but not to my seedbox's, for some reason (I know I'm probably not setting things up correctly).

Since autobrr can send torrents to my seedbox (I've tested this with other trackers and it works great), I was wondering if I could use autobrr to send to my seedbox torrents that have been fetched/approved by my script. I know that I can add custom scripts in autobrr (Filters->Actions), but I can't use Filters without an indexer and I don't know if this would work for my case. I think I can add the tracker API to Jackett and add a torznab indexer from Jackett on autobrr, but for now I just want to know if there's anything I can do without Jackett/Prowlarr.

Is there an autobrr API I could use in my script?

**Comment by KyleSanderson** [comment]
There's a mock indexer that's in-product if you really wanted to develop a custom pipeline like this. Alternatively you could have a RSS proxy from the API, then setup autobrr to hit your now customized feed.

**Reply by ikki4** [reply]
Can autobrr read a local (xml) RSS feed (read a file, not a url)?
